src/components/DownloadComponents/DownloadReleases/DownloadTable/index.tsx

```diff
@@ -63,7 +64,13 @@ const DownloadTable = ({ nodeReleasesData }: Props): JSX.Element => (
                     <a href={releaseDownloadLink}>{version}</a>
                   )}
                 </td>
-                <td>{status || ''}</td>
+                <td>
+                  <FormattedMessage
+                    id={`components.downloadTable.status.${toCamelCaseStatus(
```

Comment:
I think instead of doing this, you should have an "enum" (mapping) of the possible combination of statuses and their translation IDs:

```js
const STATUS_TRANSLATION_MAP = {
  "End of Life": "components.downloadTable.status.endOfLife",
  ...
}
```

That would remove the need of adding that helper :)